src/screens/MyListsScreen.tsx

```diff
@@ -46,11 +47,36 @@ class MyListsScreen extends React.Component<Props, State> {
     });
   }
 
-  componentDidMount() {
+  async componentDidMount() {
     this.refresh();
     this._unsubscribe = this.props.navigation.addListener('focus', () => {
       this.fetchLists();
     });
+    Audio.setAudioModeAsync({
+      allowsRecordingIOS: false,
+      interruptionModeIOS: Audio.INTERRUPTION_MODE_IOS_DO_NOT_MIX,
+      playsInSilentModeIOS: true,
+      interruptionModeAndroid: Audio.INTERRUPTION_MODE_ANDROID_DUCK_OTHERS,
+      shouldDuckAndroid: true,
+      staysActiveInBackground: true,
+      playThroughEarpieceAndroid: true,
+    });
+
+    this.sound = new Audio.Sound();
+
+    const status = {
+      shouldPlay: true,
+    };
+
+    this.sound.loadAsync(
+      require('../../assets/sounds/bubble.mp3'),
+      status,
+      true
+    );
+  }
+
+  playSound() {
+    this.sound.playAsync();
```

Comment:
```suggestion

```

Reply:
You are right, I had it there when I was trying to assign the sound to a button and just forgot to remove it. Thanks